### ISSUE TITLE
linter-swagger throwing error

### ISSUE THREAD

**Issue by fgysin** [AtomLinter/linter-swagger#104]
I installed tried to use Atom to work on OpenAPI spec files, but couldn't get it to work...
Whenever I work on my YAML spec files the linter doesn't show any errors.

Atom version: 1.38 x64

**Packages** I have installed:
- linter 2.3.0
- linter-swagger 0.6.1
- linter-ui-default 1.7.1
- linter-javac (to check if linter is working at all)
- intentions, busy-signal (both seem to be dependencies of some of the above)

I was able to verify that linting works generally: I created a small Java file and any errors in there are recognised as expected.

Here is the OpenAPI file that I tried (this is the complete file):
```
openapi: 3.0.2
info:
  title: Foobar Service
  description: Some description
  version: 0.0.1
  contact:
    name: Foobar
    url: https://foobar.com
    email: foo@bar.com
```
--> There is plenty of stuff missing for this to be valid OAS3, so I'd expect the linter to trigger somehow.

As I have read in other places that sometimes the linter wasn't triggering because it didn't recognise the fiel as Swagger/OAS I have also tried to add `swagger: 2.0`, or add single or double quotation marks for the `openapi` field - to no avail....

What can I do to make linter-swagger work?

**Comment by adlerfaulkner** [comment]
+1
